src/planet-a/icosahedron.h

```diff
@@ -91,31 +91,44 @@ inline constexpr std::array<std::array<uint8_t, 3>, 20> gc_icoNeighbors
 /**
  * @brief Icosahedron Minimum Edge Length vs Subdiv Levels, radius = 1.0
  */
-inline constexpr std::array<float, 10> const gc_icoMinEdgeVsLevel
+inline constexpr std::array<float, 24> const gc_icoMinEdgeVsLevel
 {
-    1.05146222e+0f,  5.46533058e-1f,  2.75904484e-1f,  1.38283174e-1f,  6.91829904e-2f,
-    3.45966718e-2f,  1.72989830e-2f,  8.64957239e-3f,  4.32479631e-3f,  2.16239942e-3f
+    1.05146222e+0f,  5.46533058e-1f,  2.75904484e-1f,  1.38283174e-1f,
+    6.91829904e-2f,  3.45966718e-2f,  1.72989830e-2f,  8.64957239e-3f,
+    4.32479631e-3f,  2.16239942e-3f,  1.08119987e-3f,  5.40599953e-4f,
+    2.70299979e-4f,  1.35149990e-4f,  6.75749950e-5f,  3.37874975e-5f,
+    1.68937487e-5f,  8.44687437e-6f,  4.22343719e-6f,  2.11171859e-6f,
+    1.05585930e-6f,  5.27929648e-7f,  2.63964824e-7f,  1.31982412e-7f
 };
 
 /**
  * @brief Icosahedron Maximum Edge Length vs Subdiv Levels, radius = 1.0
  */
-inline constexpr std::array<float, 10> const gc_icoMaxEdgeVsLevel
-{
-    1.05146222e+0f,  6.18033989e-1f,  3.24919696e-1f,  1.64647160e-1f,  8.26039665e-2f,
-    4.13372560e-2f,  2.06730441e-2f,  1.03370743e-2f,  5.16860619e-3f,  2.58431173e-3f
-};
+inline constexpr std::array<float, 24> const gc_icoMaxEdgeVsLevel
+{{
```

Comment:
double brace init here? that seems like maybe a typo?

Reply:
I don't remember the exact details, but this is required for some of the other tables in this file for arrays of structs. This is harmless afaik